src/utils/git.ts

```diff
@@ -21,15 +51,25 @@ export function getGitRepoAndBranch() {
       normalizedRepositoryName = repositoryName.endsWith(".git")
         ? repositoryName?.slice(0, -DOT_GIT_SUFFIX_LENGTH)
         : repositoryName;
+      const defaultBranch = await getDefaultBranch(repository.rootUri.fsPath);
+      if (!defaultBranch) {
+        showCannotGetDefaultBranchMessage();
```

Comment:
whats the difference between this error and the one we show in `getDefaultBranch`?

Reply:
here its a general message to inform the user that we cannot find the default branch for some reason and the consequences of that.
In case there is an error when executing git show remote origin there is a second message showing this error
<img width="833" alt="image" src="https://github.com/env0/vscode-env0/assets/33559192/5fe6f2ae-26e1-4ffb-b5fa-8f5180099182">